crm_phonecall_timesheet/views/crm_phonecall_view.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="crm_case_inbound_phone_tree_view" model="ir.ui.view">
+    <field name="name">CRM - Phone Calls with analytic account tree</field>
+    <field name="model">crm.phonecall</field>
+    <field name="inherit_id" ref="crm_phonecall.crm_case_phone_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="date" position="after">
+            <field name="duration" widget="float_time" />
+        </field>
+        <field name="opportunity_id" position="after">
+            <button name="button_end_call"
+                    string="Stop"
+                    type="object"
+                    icon="fa-times-circle text-danger"
```

Comment:
Don't put it on red and use `fa-stop-circle` the same as in project_timesheet_time_control.